samples/detect.v2beta1.js

```diff
@@ -0,0 +1,728 @@
+/**
+ * Copyright 2017, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const projectId = process.env.GCLOUD_PROJECT;
+const sessionId = require('uuid/v1')();
+const common = require('@google-cloud/common');
+const structjson = require('./structjson.js');
+
+function createKnowledgeBase(projectId, displayName) {
+  // [START dialogflow_create_knowledge_base]
+  // Imports the Google Cloud DialogFlow library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow client.
+  var client = new dialogflow.KnowledgeBasesClient();
+
```

Comment:
There usually is a few commented lines such as these in every sample function:

  /**
   * TODO(developer): Uncomment the following lines before running the sample.
   */
  // const projectId = 'The project id, e.g. 12345678';
  // const displayName = 'The Display Name';

You probably want to add lines like these to all the sample functions (Rebecca taught me this)

Reply:
I see the TODOs, marking resolved